recipes/lm/sft/README.md

```diff
@@ -0,0 +1,141 @@
+# Language Model Supervised Fine-Tuning (SFT)
+
+This directory contains the implementation for supervised fine-tuning of language models using fairseq2. The SFT recipe allows you to fine-tune pre-trained language models on instruction-following datasets.
+
+## Overview
+
+The SFT recipe is designed to finetune language models to follow instructions by learning from prompt-response pairs.
+
+This recipe supports both Llama and Qwen model families.
+
+## Key Features
+
+- **Flexible Data Loading**: Support for both local files and Hugging Face Hub datasets
+- **Chat Mode**: Built-in support for chat templates (compatible with Hugging Face chat templates)
+- **Dynamic Batching**: Static batching or length-based batching for efficient training
+- **Distributed Training**: Full support for multi-GPU and multi-node training
+
+## Quick Start
+
+### Basic Usage
+
+```bash
+# Run with a pre-configured Llama setup
+python -m fairseq2.recipes.lm.sft --config-name recipes/lm/sft/configs/llama3_2_1b_gsm8k.yaml
+
+# Run with a pre-configured Qwen setup
+python -m fairseq2.recipes.lm.sft --config-name recipes/lm/sft/configs/qwen3_4b_gsm8k.yaml
+
+# Run with custom config file")
+python -m fairseq2.recipes.lm.sft --config-file path/to/your/config.yaml
```

Comment:
I think you don't need "fairseq2" here (and in other examples) right? It ideally should work with

```
python -m recipes.lm.sft ...
```